tasks/db_deploy/test/manual_tests/sql/orca_schema_v3/remove.sql

```diff
@@ -1,11 +1,14 @@
+-- Drop our changes for V3
 ALTER TABLE orca.recovery_file
-	DROP COLUMN multipart_chunksize_mb
-GO
+	DROP COLUMN IF EXISTS multipart_chunksize_mb
+
+-- Reset the versions table
 DELETE FROM orca.schema_versions
     WHERE version_id=3
-GO
-UPDATE orca.schema_versions
-    SET is_latest=true
-    WHERE version_id=2
-GO
-COMMIT;
+-- Upsert the old version
+INSERT INTO schema_versions
```

Comment:
should this be `INSERT INTO orca.schema_versions`?

Reply:
Fixed